corehq/apps/geospatial/templates/map_visualization.html

```diff
@@ -57,6 +57,9 @@ <h4 class="modal-title">{% trans "Cases Missing GPS Data" %}</h4>
                                 <tr><td data-bind="html: link"></td></tr>
                             </tbody>
                         </table>
+                        <a class="btn btn-danger" target="_blank" href="{% url 'gps_capture' domain %}?data_type=case">
```

Comment:
@zandre-eng why is it using the `btn-danger` class?

Reply:
@Charl1996 I was following the same design as the button that opens this modal. Should we rather make this a `button-default`?

Reply:
I'd personally vote for that, because the red implies a dangerous action, don't you think?

Reply:
I agree, that makes sense. Should we then change the colour of the "Missing Cases" button too? I think it might be good to show this as a non-dangerous action as well.

Reply:
Do we need this button now? I think earlier we didn't have the link to GPS capture page in sidebar, but now we do.
Can check with Mary once about this, if we need this button.

Reply:
It does look like its simply to give a way to reach to this page quickly, since they are already looking at cases missing gps data.
Ya, it does look like color could be other than red, or kept the same as well.

Reply:
I had a sync with Mary on this, and it was agreed that the modal for this would be removed and replaced with a button that takes the user to the "GPS Capture" page. I've replaced this button's class to `btn-default` to correctly reflect it as a non-dangerous action ([c1d80c6](https://github.com/dimagi/commcare-hq/pull/33420/commits/c1d80c60bfdcb467cc175c84f3a64cce45516fb3)).